lib/linguist/languages.yml

```diff
@@ -449,6 +449,14 @@ Cap'n Proto:
   - .capnp
   ace_mode: text
 
+CartoCSS:
+  aliases:
+  - Carto
+  extensions:
+  - .mss
+  ace_mode: text
```

Comment:
I only know about a [grammar for Atom](https://github.com/yohanboniface/carto-atom), that [looks pretty good](https://lightshow.githubapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_url=https%3A%2F%2Fraw.githubusercontent.com%2Fyohanboniface%2Fcarto-atom%2Fmaster%2Fgrammars%2Fcarto.json&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fraw.githubusercontent.com%2Fgravitystorm%2Fopenstreetmap-carto%2Fmaster%2Flandcover.mss&code=).